doc/manuals/admin/build_source.md

```diff
@@ -85,16 +87,32 @@ for details. Recommended version is 3.6 (although 3.2 and 3.4 should also work f
 
 The Orion Context Broker comes with a suite of functional, valgrind and end-to-end tests that you can also run, following the following procedure (optional):
 
+* To install mongodb-org-shell using yum, create a /etc/yum.repos.d/mongodb.repo file.
+
+        [mongodb-org-3.6]
+        name=MongoDB Repository
+        baseurl=https://repo.mongodb.org/yum/redhat/$releasever/mongodb-org/3.6/x86_64/
+        gpgcheck=1
+        enabled=1
+        gpgkey=https://www.mongodb.org/static/pgp/server-3.6.asc
+
```

Comment:
Apart from the mongodb.repo, maybe the yum command to actually install the mongo shell should be added?

Reply:
Sorry, I made an editing mistake. I added 'mongodb-org-shell' in line 101 again.

Reply:
Fixed in 6039fe1